kots-preflight.yaml

```diff
@@ -63,3 +63,27 @@ spec:
               message: The cluster should contain at least 4 cores.
           - pass:
               message: There are at least 4 cores in the cluster.
+        checkName: Required Node Available Disk Space
+        #The below outcomes are intentionally set to very high values as these should be adjusted based on the needs of your application.
+        strict : true
```

Comment:
I would not make it `strict`. This will for sure always fail, and we actually want to have some success from the start.

Reply:
What would reasonable values be here then? From my initial pr comment and from the comment in line here, I purposefully set these high so they would fail to encourage people to change them (though I did allude to my expectation of this being a contentious direction)